Gemfile

```diff
@@ -73,6 +73,7 @@ group :development do
   gem 'rubocop', '~> 1.30', require: false
   gem 'rubocop-rails', '~> 2.14.2', require: false
   gem 'rubocop-rspec', '~> 2.11.1', require: false
+  gem 'strong_migrations', '~> 1.0.0'
```

Comment:
This here looks a bit weird. Since we have `strong_migrations` listed as in the dev/test joint group, we should remove it from here.

Reply:
We should remove this line :)

Reply:
sorry, I thought I removed it 😬